02_problem_statement_and_structure.md

```diff
@@ -147,15 +148,18 @@ The `fuels dev` command does three things:
 4. Saves their deployed IDs of contracts to: {output_in_config_file}/contract-ids.json
 
 # Important Notes
+
 Some of the important points to note are:
-* In the `fuels.config.ts` file, we have specified the contracts and output fields, but we could have also specified the workspace field instead of the contracts field.
-For more information on the workspace field, you can visit [here](https://docs.fuel.network/docs/forc/workspaces/#workspaces).
-* Here we will be using Fuels CLI which wraps forc. `forc` commands like `forc build`, `forc deploy`, etc. can also be used directly for building and deploying contracts.
-* Some other important Fuels CLI commands are 
-  * `fuels build`: This command builds all Sway programs under your contracts using forc.
-  * `fuels deploy`: This command deploys all Sway contracts and saves their deployed IDs to: `{output_in_config_file}/contract-ids.json`. It is recommended to use the `fuels deploy` command only when deploying contracts to a local node. If you are deploying contracts to a live network like the Testnet, it is recommended to use the `forc deploy` command instead.
+
+- In the `fuels.config.ts` file, we have specified the contracts and output fields, but we could have also specified the workspace field instead of the contracts field.
+  For more information on the workspace field, you can visit [here](https://docs.fuel.network/docs/forc/workspaces/#workspaces).
+- Here we will be using Fuels CLI which wraps forc. `forc` commands like `forc build`, `forc deploy`, etc. can also be used directly for building and deploying contracts.
+- Some other important Fuels CLI commands are
+  - `fuels build`: This command builds all Sway programs under your contracts using forc.
+  - `fuels deploy`: This command deploys all Sway contracts and saves their deployed IDs to: `{output_in_config_file}/contract-ids.json`. It is recommended to use the `fuels deploy` command only when deploying contracts to a local node. If you are deploying contracts to a live network like the Testnet, it is recommended to use the `forc deploy` command instead.
 
 Here is the [link](https://docs.fuel.network/docs/nightly/fuels-ts/fuels-cli/commands/) to the official documentation for the Fuels CLI commands.
 
 # Output
-TODO - Explain a bit about the output generated by the Fuels CLI here. The details are covered in the frontend section.
+
+In this command `npx fuels@0.82.0 init --contracts ./contracts/TokenTrack/ --output ./frontend/src/sway-contracts-api`, the path we specified with the output flag will have the typed interface files for static type checking and error prevention in accordance with the contract which is helpful in frontend development.
```

Comment:
refer to file `output` field of the file here. We have explained the contents above